spec/imsc-hrm.html

```diff
@@ -198,9 +198,21 @@
 
     <p>Furthermore, the model is not intended to constrain readability complexity.</p>
   </section>
-
   <section id='sotd'>
-   
+    <p>For this specification to exit the CR stage, at least 1 of each, and at least 2 of one, of the following
+    independent factors of verifications SHALL be demonstrated:</p>
+      <ul>
+        <li>content producing implementation</li>
+        <li>validating implementation</li>
+      </ul>
```

Comment:
In view of the algorithm specified by the specification, if there were two validating implementations and no content producing implementations, I think that should be adequate to exit CR. I don't believe it is symmetrical the other way though: 2 content producing implementations would not be adequate.

```suggestion
    <p>For this specification to exit the CR stage, one of the following minimum
    combinations of independent factors of verifications SHALL be demonstrated:</p>
      <ul>
        <li>at least 1 content producing implementation and at least 1 validating implementation OR</li>
        <li>at least 2 independent validating implementations</li>
      </ul>
```